python3-tryton.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "python3-tryton",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"tryton[calendar]==7.2.0\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/5f/25/4a397bbe70a07819b438e3948e2796ca9086b7aa1eb645acaee25d3074fa/GooCalendar-0.8.0-py3-none-any.whl",
+            "sha256": "ac39247add2f06190081d6e617e3ec75ac3f4a41ddaf0206f6ed7dfe24d9b6d5"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/1c/41/91955188e97c7b85fbaac6bbf4e33de028899e0aa31bdce99b1afe2eeb17/pycairo-1.26.0.tar.gz",
+            "sha256": "2dddd0a874fbddb21e14acd9b955881ee1dc6e63b9c549a192d613a907f9cbeb"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/f9/9e/6bab1ed3bd878f0912d9a0600c21f3d88bab0e8a8d4c3ce50f5cf336faaf/pygobject-3.48.2.tar.gz",
```

Comment:
isnt this already in the runtime?

Reply:
It is flatpak-builder-tools that generate it.

Reply:
did you tell it to explicitly use the GNOME 46 runtime?

Reply:
nevermind. let's leave it at that.

Reply:
The yml file set the runtime to 46.